site/docs/4.2/components/modal.md

```diff
@@ -234,9 +307,36 @@ Add `.modal-dialog-centered` to `.modal-dialog` to vertically center the modal.
   </div>
 </div>
 
+<div id="exampleModalCenteredScrollable" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="exampleModalCenteredScrollableTitle" aria-hidden="true">
```

Comment:
It seems this one doesn't have enough content to show as scrollable.

Reply:
It is intentional for reproduce both centering and scrollable. If too much content, nobody can see centered modal.

Reply:
I'm not sure how useful this is then. But if everyone else thinks it's OK, that's fine I guess.